openapi/schemas/processes-job-management/processes-type.yaml

```diff
@@ -0,0 +1,5 @@
+type: string
+nullable: false
+enum:
+  - process
+  - openeo
```

Comment:
Should this be made a simple string with examples?
Do we want to create the same situation as `statusCode` needing an override because of the new `created` status?

If the long term use is to have job management available for an OGC API, maybe it would be better to define a Requirements class that say for openEO, `type: openeo` MUST be used, and `process` for *OGC API - Processes*. A "long" Coverage processing could then easily define their own requirement class with `type: coverage`, without causing invalid JSON schema definitions.